### ISSUE TITLE
Atualizar README 

### ISSUE THREAD

**Issue by jonathantvrs** [jackson541/API-Boletim-Covid#16]
Atualizar com informações sobre o projeto como:

Como rodar
Status de Integração Contínua
Contribuidores
Dependências do Projeto

**Comment by jackson541** [comment]
No "como rodar" ainda faltam os comandos pra criar as tabelas no bando de dados.

**Comment by jonathantvrs** [comment]
Verdade!

**Comment by jackson541** [comment]
Fazer um merge da 16_atualizar_readme direto para a master não vai resultar em um conflito quando for realizar um merge da develop para a master?

**Comment by jonathantvrs** [comment]
É só fazer um merge da master com a develop 
Me esqueci da develop, foi a pressa aqui, mas vou consertar

**Comment by jonathantvrs** [comment]
Eu criei a 16 da master, por isso foi direto.

**Comment by jackson541** [comment]
É porque já fiz o crud de caso na branch dele com base na develop, mas falta atualizar algumas coisas, e já realizei o merge com a develop. Por isso, pensei que poderia dar zebra.

**Comment by jonathantvrs** [comment]
Pronto! Resolvido

**Comment by jonathantvrs** [comment]
Me diz uma coisa, sabe resolver conflitos na mão?

**Comment by jackson541** [comment]
Sim, pelo menos com o vscode. Ele mostra quais são as linhas em conflito e a pessoa pode escolher qual vai ficar

**Comment by jonathantvrs** [comment]
Show

**Comment by jackson541** [comment]
Agora falta falar quais são especificações do postgres para rodar (nome da tabela, login e senha) e a url para encontrar o swagger.

**Comment by jonathantvrs** [comment]
Fique a vontade para adicionar
